src/instrumentation/knex.js

```diff
@@ -0,0 +1,66 @@
+
+const shimmer = require('shimmer')
+// Modified from https://github.com/elastic/apm-agent-nodejs
+
+// BSD 2-Clause License
+
+// Copyright (c) 2012, Matt Robenolt
+// Copyright (c) 2013-2014, Thomas Watson Steen and Elasticsearch BV
+// Copyright (c) 2015-2020, Elasticsearch BV
+// All rights reserved.
+
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions are met:
+
+// * Redistributions of source code must retain the above copyright notice, this
+//   list of conditions and the following disclaimer.
+
+// * Redistributions in binary form must reproduce the above copyright notice,
+//   this list of conditions and the following disclaimer in the documentation
+//   and/or other materials provided with the distribution.
+
+const instrumentKnex = (knex, { logger, buffer, enabled }) => {
+  if (!enabled) return knex
+
+  logger.info('Shimming knex')
+
+  if (knex.client) {
+    logger.debug('shimming Knex.client.runner')
+    shimmer.wrap(knex.client, 'runner', wrapRunner)
+  } else {
+    logger.debug('could not shim Knex')
```

Comment:
Please update all logger statements to follow the same pattern as per the following example: https://github.com/mojaloop/central-ledger/blob/master/src/shared/setup.js#L253